hathor/cli/run_node.py

```diff
@@ -49,6 +49,10 @@ def create_parser(self) -> ArgumentParser:
         parser.add_argument('--recursion-limit', type=int, help='Set python recursion limit')
         parser.add_argument('--allow-mining-without-peers', action='store_true', help='Allow mining without peers')
         parser.add_argument('--min-block-weight', type=int, help='Minimum weight for blocks')
+        parser.add_argument('--x-fast-init-beta', action='store_true',
+                            help='Execute a fast initialization, which skips some transaction verifications. '
+                            'This is still a beta feature because there are some issues when using both '
+                            'initializations.')
```

Comment:
What do you mean by "both initializations"?

Reply:
Fast initialization and full verification. But actually the problem is with sudden crashes, I will improve this help text.

Reply:
Updated the help text, can you see if it's good?

Reply:
Proposed a new text.